lib/tasks/jobs.rake

```diff
@@ -0,0 +1,17 @@
+namespace :db do
```

Comment:
I know the example on the gem puts this task under the `db` namespace but I think it would make more sense to have this job namespaced under the existing `jobs` namespace and call the job `schedule`. e.g. 

```ruby
namespace :jobs do
  task schedule: :environment do
  ...
  end
end
```